disperser/dataapi/docs/swagger.yaml

```diff
@@ -138,6 +138,19 @@ definitions:
       total_unsigned_batches:
         type: integer
     type: object
+  dataapi.OperatorPortCheckResponse:
+    properties:
+      disperser_online:
+        type: boolean
+      disperser_socket:
```

Comment:
"dispersal_socket", e.g. the naming for port https://github.com/Layr-Labs/eigenda/blob/cb32efd4fabca4e7db2d1b56113e0fa5e39a654c/node/config.go#L43